resources/compass/charts/kyma-environment-broker/templates/secrets.yaml

```diff
@@ -47,11 +47,20 @@ metadata:
 apiVersion: v1
 kind: Secret
 metadata:
-  name: "{{ .Values.lms.secretName}}"
+  name: "{{ .Values.lms.secretName }}"
   labels:
 {{ include "kyma-env-broker.labels" . | indent 4 }}
 type: Opaque
 data:
   token: {{ .Values.lms.token | b64enc | quote }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "{{ .Values.ias.secretName }}"
+  labels: {{ include "kyma-env-broker.labels" . | nindent 4 }}
```

Comment:
on compass repo. you have provided labels in [that](https://github.com/kyma-incubator/compass/commit/caf93a4930abc70e2e4d68f8037e8eff28b16a91#diff-815aa8e14702c762cafeb98af57966a9R44) way, maybe let's have it consistent

Reply:
discussed.